pkg/functionconfig/types.go

```diff
@@ -577,6 +577,13 @@ func NewConfig() *Config {
 		Meta: Meta{
 			Namespace: "default",
 		},
+		Spec: Spec{Triggers: map[string]Trigger{
+			"some-trigger": {
+				Kind:       "http",
+				MaxWorkers: 1,
+				Name:       "some-trigger",
+			},
+		}},
```

Comment:
This function is not only used in tests, but also in nuctl - this can cause some unexpected behavior when working with nuctl.